lib/elastomer/client/native_delete_by_query.rb

```diff
@@ -38,6 +38,9 @@ def initialize(client, query, parameters)
           raise IncompatibleVersionException, "Elasticsearch '#{client.version}' does not support _delete_by_query"
         end
 
+        # COMPATIBILITY: read_timeout param is invalid, must remove at last moment
+        parameters.select! { |k, _| k.to_sym != :read_timeout }
```

Comment:
As mentioned in chat, `#reject!` is more idiomatic.

Reply:
same here